tomcat/src/main/java/nextstep/jwp/model/Content.java

```diff
@@ -0,0 +1,36 @@
+package nextstep.jwp.model;
+
+import nextstep.jwp.exception.NotFoundContentTypeException;
+
+import java.util.Arrays;
+import java.util.Objects;
+
+public enum Content {
+
+    HTML("html", "text/html"),
+    CSS("css", "text/css"),
+    JS("js", "application/javascript"),
+    ICO("ico", "image/x-icon"),
+    ;
+
+    private final String extension;
+    private final String type;
+
+    Content(String extension, String type) {
+        this.extension = extension;
+        this.type = type;
+    }
+
+    public static String getType(String extension) {
+        Objects.requireNonNull(extension);
+        Content content = Arrays.stream(values())
+                .filter(it -> it.extension.equals(extension))
+                .findAny()
+                .orElseThrow(() -> new NotFoundContentTypeException("ContentType을 찾을 수 없습니다."));
+        return content.type;
+    }
```

Comment:
```suggestion
    public static String getType(String extension) {
        Objects.requireNonNull(extension);
        return Arrays.stream(values())
                .filter(it -> it.extension.equals(extension))
                .map(it -> it.type)
                .findAny()
                .orElseThrow(() -> new NotFoundContentTypeException("ContentType을 찾을 수 없습니다."));
    }
```
type으로 바로 반환해줘도 될 것 같아요!

Reply:
Enum Content클래스가 아닌 String타입인 type변수를 반환하도록 한 메서드여서 바로 반환해주지 않았었는데, 
```java
    public static String getType(String extension) {
        Objects.requireNonNull(extension);
        return Arrays.stream(values())
                .filter(it -> it.extension.equals(extension))
                .findAny()
                .orElseThrow(() -> new NotFoundContentTypeException("ContentType을 찾을 수 없습니다."))
                .type;
    }
```
가 더 깔끔할 것 같아요! 이렇게 수정해보겠습니다.